helpers/translateCategory.js

```diff
@@ -0,0 +1,30 @@
+export function translateCategory(categoryInEnglish) {
+  switch (categoryInEnglish) {
+    case "fruits&vegetables":
+      return "Obst & Gemüse";
+
+    case "homemade":
+      return "Selbstgemacht";
+
+    case "pastry":
+      return "Backwaren";
+
+    case "dairy":
+      return "Käse, Eier & Molkerei";
+
+    case "frozen":
+      return "Tiefkühlkost";
+
+    case "sweet&salty":
+      return "Süßes & Salziges";
+
+    case "beverage":
+      return "Getränke";
+
+    case "meat&fish":
+      return "Fleisch & Fisch";
+
+    default:
+      return "Da ist etwas falsch gelaufen...";
+  }
```

Comment:
This function is a bit of an overkill. You can either save the german translation directly in your dataset e.g. have category: "pastry" and categoryGerman: "Backwaren", or you use an object here. The keys of the object are the english names, e.g. "pastry" and the translation is the key. By this you can get the translation by using the `[]` notation of the Object:
```js
const translationMap = {
  "fruits&vegetables": "Obst & Gemüse",
  "homemade": "Selbstgemacht",
  ...
}

console.log(translationMap["homemade"]) // logs Selbstgemacht

```

Reply:
yes, changed this one as well 🦦 looks a lot slimmer now